tools.json

```diff
@@ -221,6 +221,22 @@
       },
       "test": "phpDocumentor list"
     },
+    {
+      "name": "phpbench",
+      "summary": "PHP Benchmarking framework",
+      "website": "https://github.com/phpbench/phpbench",
+      "command": {
+        "file-download": {
+          "url": "https://phpbench.github.io/phpbench/phpbench.phar.pubkey",
+          "file": "/usr/local/bin/phpbench.phar.asc"
```

Comment:
I think [phpbench test fails](https://travis-ci.org/jakzal/phpqa/jobs/418200379#L1839) because of the `.asc` extension (should be `.pubkey`).

Reply:
Yes, I was looking into it, then ran out of time, weill fix to get this moving, ty.